apps/lfx-one/src/app/modules/committees/committee-view/committee-view.component.html

```
@@ -33,374 +33,362 @@ <h2 class="text-lg font-semibold text-gray-900 mb-2">Something Went Wrong</h2>
+                } @else if (subGroups().length === 1) {
+                  <!-- Single sub-group: inline link, no dropdown -->
+                  <button
+                    class="flex items-center gap-1.5 text-xs font-medium text-blue-600 hover:text-blue-800 transition-colors cursor-pointer"
+                    (click)="navigateToSubGroup(subGroups()[0])"
+                    data-testid="committee-view-sub-group-link">
+                    <i class="fa-light fa-sitemap text-[11px]"></i>
+                    <span>Sub-Group:</span>
+                    <span>{{ subGroups()[0].name }}</span>
+                    <i class="fa-light fa-chevron-right text-[10px]"></i>
+                  </button>
+                } @else if (subGroups().length > 1) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)